disasterinfosite/static/css/app.css

```diff
@@ -167,6 +173,14 @@ label {
   padding: 15px;
 }
 
+.information-container a {
```

Comment:
I'm not sure i like this, actually- the orange + dark green almost have too much contrast. Maybe we could bold them instead or something.